src/drivers/sf0x/sf0x.cpp

```diff
@@ -262,7 +262,7 @@ SF0X::init()
 {
 	/* do regular cdev init */
 	if (CDev::init() != OK) {
-		goto out;
```

Comment:
The fds were not being closed in the existing code and my code kept the same behaviour. I'm submitting a commit now to fix it to always close.